internal/cmd/update/command.go

```diff
@@ -74,6 +80,21 @@ func NewClient(cliName string, disableUpdateCheck bool) update.Client {
 }
 
 func (c *command) update(cmd *cobra.Command, _ []string) error {
+	if c.Config.DisableUpdates {
+		message := "updates are disabled for this binary"
+		if isHomebrew() {
+			return errors.NewErrorWithSuggestions(
+				message,
+				fmt.Sprintf("If installed with Homebrew, run `brew upgrade %s`.", homebrewFormula),
+			)
+		}
+
+		return errors.NewErrorWithSuggestions(
+			message,
+			"If installed with Confluent Platform, you may install an independent, updated version of the CLI, as long as it maintains compatibility: https://docs.confluent.io/platform/current/installation/versions-interoperability.html#confluent-cli",
```

Comment:
```suggestion
			"Use a package manager to update the binary, if applicable. Otherwise, consider deleting this binary and re-installing a newer version.",
```